### PR TITLE
Add `oauth_tokens` to AuthkitAuthenticationResponse

### PR DESCRIPTION
**Pull request by jonatascastro12**
## Description

Created a new `OAuthTokens` model to represent OAuth credentials, including provider, access token, refresh token, expiration, and scopes. Integrated this new model into the `AuthenticationResponse` class, adding an optional field for OAuth tokens.

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

**Review by mattgd — COMMENTED** [review]
Looks good, just some small feedback.

File: workos/types/user_management/authentication_response.py [hunk 1]
```diff
@@ -28,6 +29,7 @@ class AuthenticationResponse(_AuthenticationResponseBase):
     impersonator: Optional[Impersonator] = None
     organization_id: Optional[str] = None
     user: User
+    oauth_tokens: Optional[OAuthTokens] = None
```
[the remark this answers is not in the document]
Comment (jonatascastro12) [reply workos/types/user_management/authentication_response.py:32]: Just for `AuthenticateWithCode` - so I guess we should split this type apart?

[the remark this answers is not in the document]
Comment (jonatascastro12) [reply workos/types/user_management/authentication_response.py:32]: @mattgd I'm wondering if I change the response, would it be considered a breaking change?

[the remark this answers is not in the document]
Comment (jonatascastro12) [reply workos/types/user_management/authentication_response.py]: Never mind, I found that I had the new field to the `AuthKitAuthenticationResponse`

**Review by mattgd — APPROVED** [review]

File: workos/types/user_management/authentication_response.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattgd) [reply workos/types/user_management/authentication_response.py]: I forgot about that type, nice!

**Pull request merged**